tests/system/marathon_common_tests.py

```diff
@@ -1133,3 +1133,25 @@ def http_output_check():
         assert 'Relay from {}'.format(relay_app["id"]) in output
 
     http_output_check()
+
+
+@shakedown.dcos_1_11
+def test_ipv6_healthcheck():
```

Comment:
I would pass the fixture directly to the test e.g. `def test_ipv6_healthcheck(docker_ipv6_network_fixture)`